<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginsList</class>
 <widget class="QWidget" name="PluginsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Application Extensions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="allowAppPlugins">
         <property name="text">
          <string>Allow Application Extensions to be loaded</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="verticalFrame">
         <layout class="QVBoxLayout" name="appExtensionFrame">
          <item>
           <widget class="QListWidget" name="list">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="uniformItemSizes">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="butSettings">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Settings</string>
              </property>
              <property name="icon">
               <iconset resource="../data/icons.qrc">
                <normaloff>:/icons/preferences/preferences-desktop.png</normaloff>:/icons/preferences/preferences-desktop.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>WebKit Plugins</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;b&gt;Click To Flash Plugin&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Click To Flash is a plugin which blocks auto loading of Flash content at page. You can always load it manually by clicking on the Flash play icon.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" rowspan="3">
          <widget class="QTreeWidget" name="whitelist">
           <column>
            <property name="text">
             <string>Whitelist</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="add">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="remove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="allowClick2Flash">
         <property name="text">
          <string>Allow Click To Flash</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../data/icons.qrc"/>
 </resources>
 <connections/>
</ui>
